File: Prob02/Prob02.py
def main():
    with open("Prob02.in.txt") as f:
        myList = [line.rstrip('\n') for line in f]
        n = int(myList[0])

        currLine = None
        for i in range(1, n + 1):
            currLine = myList[i].split()
            word = currLine[0]
            letter = int(currLine[1])
            for j in range(len(word)):
                if j != letter:
                    print(word[j], end = "")
            print("")

if __name__ == "__main__": main()

File: Prob02.java
import java.io.*;

public class Prob02 {

    private static final String INPUT_FILE_NAME = "Prob02.in.txt";

    public static void main(String[] args) {
        try {
            File input = new File(INPUT_FILE_NAME);
            FileReader fr = new FileReader(input);
            BufferedReader br = new BufferedReader(fr);

            int n = Integer.parseInt(br.readLine());

            String currLine = null;
            for(int i = 0; i < n; i++) {
                currLine = br.readLine();
                String[] parts = currLine.split(" ");
                int letter = Integer.parseInt(parts[1]);
                for(int j = 0; j < parts[0].length(); j++) {
                    if(j != letter) {
                        System.out.print(parts[0].charAt(j));
                    }
                }
                System.out.println();
            }
            br.close();
            fr.close();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}
